test/e2e/testing-manifests/ingress/neg-clusterip/rc.yaml

```diff
@@ -15,7 +15,7 @@ spec:
         run: hostname
     spec:
       containers:
-      - image: gcr.io/kubernetes-e2e-test-images/serve-hostname:1.1
+      - image: k8s.gcr.io/e2e-test-images/serve-hostname:1.1
```

Comment:
same as above for the serve-hostname image, and same for below, there are quite a few instances.